src/TreeCounter/App.svelte

```diff
@@ -0,0 +1,501 @@
+                            var el = document.createElement("div");
+                            var treeCount = document.createTextNode(
+                                contribution.properties.treeCount
+                            );
+
+                            el.appendChild(treeCount);
+
+                            var svgNS = "http://www.w3.org/2000/svg";
+                            var tree = document.createElementNS(svgNS, "svg");
+
+                            tree.setAttributeNS(null, "width", 10.5);
+                            tree.setAttributeNS(null, "height", 12.598);
+                            tree.setAttributeNS(
+                                null,
+                                "viewBox",
+                                "0 0 10.5 12.598"
+                            );
+
+                            var pathNS = "http://www.w3.org/2000/svg";
+                            var path = document.createElementNS(pathNS, "path");
+                            path.setAttributeNS(
+                                null,
+                                "d",
+                                "M9,15.1V12.235a2.9,2.9,0,0,1-1,.175,3.033,3.033,0,0,1-2.093-5.2,3.055,3.055,0,0,1-.24-1.19,3,3,0,0,1,5.5-1.68h.167a3.7,3.7,0,1,1-1,7.252v3.5Z"
+                            );
+                            path.setAttributeNS(
+                                null,
+                                "transform",
+                                "translate(-4.75 -2.75)"
+                            );
+                            path.setAttributeNS(null, "fill", primaryColor);
+                            path.setAttributeNS(null, "stroke", "#fff");
+                            path.setAttributeNS(null, "stroke-width", 0.5);
+                            tree.appendChild(path);
+                            el.appendChild(tree);
+
+                            el.className = "marker";
+
+                            // make a marker for each feature and add to the map
+                            new mapboxgl.Marker(el, {
+                                anchor: "bottom",
+                            })
+                                .setLngLat(contribution.geometry.coordinates)
+                                .addTo(map);
+                        }
+                    });
+                }
+            });
+        });
+    };
+</script>
+
+<div
+    class="treecounter"
+    style="--primary-color: {primaryColor};--counter-background-color: {counterBGColor}; --background-color: {theme ===
+    'light'
+        ? '#fff'
+        : '#2f3336'}"
+>
+    {#await fetchProfileData}
+        <UserProfileLoader />
+    {:then data}
+        <div class="treeCounterContainer">
+            <div class="treeCounterComponent">
+                <div class="treeCounter">
+                    <div class="textContainer">
+                        <p class={`treecount ${theme === 'dark' ? "planted" : ""}`}>
+                            {community
```

Comment:
This is still not working for me using `community=false` - community is always true if set as a property.